pkg/cloudprovider/launchtemplate.go

```diff
@@ -53,14 +54,14 @@ type LaunchTemplateProvider struct {
 	sync.Mutex
 	ec2api                ec2iface.EC2API
 	amiFamily             *amifamily.Resolver
-	securityGroupProvider *SecurityGroupProvider
+	securityGroupProvider *provider.SecurityGroupProvider
```

Comment:
Worth thinking about package naming to avoid stuttering:
provider.SecurityGroupProvider // stutters the word provider twice
providers.SecurityGroup // ambiguous type, where SecurityGroup could be confused with the data object of ec2.SecurityGroup
securitygroup.Provider // no stuttering, naming flows nicely into a variable named securityGroupProvider

WDYT?
/pkg/providers/securitygroup/provider.go

Reply:
Yeah, I can make that change.